Placeeo/Controller/LocationManager.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  LocationManager.swift
+//  Placeeo
+//
+//  Created by Pasha Berger on 03.10.2021.
+//
+
+import Foundation
+import CoreLocation
+
+class LocationManager: NSObject, CLLocationManagerDelegate {
+    static let shared = LocationManager()
```

Comment:
why did you do this?

Reply:
I created this Singleton for make code safer and reduce RAM usage